types/git.d.ts

```diff
@@ -0,0 +1,338 @@
+/*!
```

Comment:
Don't see this in https://github.com/DefinitelyTyped/DefinitelyTyped so I guess we need to commit it.

Reply:
Yeah it's found within VS Code's source.